packages/web/src/pages/sign-up-page/utils/socialMediaMessages.ts

```diff
@@ -0,0 +1,15 @@
+export const messages = {
+  verificationError:
+    'Something went wrong. Please try again or verify with another account.',
+  socialMediaLoginSucess: (platform: 'twitter' | 'instagram' | 'tiktok') => {
+    let platformName
+    if (platform === 'twitter') {
+      platformName = 'Twitter'
+    } else if (platform === 'instagram') {
+      platformName = 'Instagram'
+    } else {
+      platformName = 'TikTok'
+    }
+    return `${platformName} connection successful!`
+  }
```

Comment:
I think this would be cleaner but def not necessary to change

```suggestion
  socialMediaLoginSucess: (platform: 'twitter' | 'instagram' | 'tiktok') => {
    const platformName = {
      twitter: 'Twitter',
      instagram: 'Instagram',
      tiktok: 'TikTok'
    }[platform]
    return `${platformName} connection successful!`
  }
```